Dimanaux/java-lab-iaas-2020
# Runnable web app

It is possible to run the app now.

## How to test

Run `mvn spring-boot:run`
Open http://localhost:8080/

You should see "Hello, world!"
